packer/plugin/packer-builder-azure-arm/main_test.go

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
```

Comment:
IMHO, we can leave this file out. It's cheating, in that it suggests we have tests while we don't. :-)
If you still want to keep it (after being called a cheater), change the license.

:stuck_out_tongue_winking_eye: